### DISCUSSION TITLE
Discuss Near

### DISCUSSION THREAD

**Discussion by Supeeerpower** [knightsven/near-smart-contract-rust-template#2] [Q&A] [answered]
What is the difference between Chain Abstraction and Account Abstraction?

**Comment by magicbroomstick** [comment, marked as answer]
Chain abstraction is about simplifying the user and developer experience in the blockchain space by hiding the complexities of interacting with different blockchains.
Account abstraction is a blockchain technology that allows users to use smart contracts as their accounts.
